### ISSUE TITLE
Serial1 not work

### ISSUE THREAD

**Issue by leoujz** [r-pv/rSTM32F030C8T6_ProMiniPlus#2]
Serial works, but Serial1 not work on my board, I connect PA2 PA3 as Serial1. I can almost 100% make sure that wiring and board are right. Please take a look at the issue, or give me a hint. Thanks

```
void setup() {

  Serial.begin(9600);

  Serial1.begin(9600);
}

void loop() {
  Serial.println(0);
  Serial1.println(0);
  delay(1000); 
}
```

**Comment by r-pv** [comment]
Hi, yes, I didn't set serial1, but software serial it's now available on the core 1.7, so you can use any pins you want (just look on the core examples). I was waiting until were possible to set i2c2 and spi2 on the core to set serial1, but you can use right now the software serial, I'll set serial1 ASAP though. Cheers.

**Comment by r-pv** [comment]
> 
> 
> Thanks for your job that makes STMF030C8T6 working with Arduino. I finally use the following code to access PA2 PA3
> 
> ```
> #include <HardwareSerial.h>
> 
> HardwareSerial Serial2(USART2);
> 
> void setup() {
>   Serial2.begin(9600); 
> }
> 
> void loop() {
>   Serial2.println(0);
>   delay(1000);   
> }
> ```

Awesome that you made it work, I didn't know that you can call directly the USART2 definition.
And thanks for the compliment but mostly it's the ST core and the ST guys that made it so easy to add new boards/variants.
Cheers.
